woocommerce/woocommerce-ios
# Add Install Jetpack row in Settings

Closes: #5369 

### Description
- Add `Install Jetpack` row to app settings screen behing `jetpackConnectionPackageSupport` feature flag. The tap action will be added later when working on issue #5370 
- Add tests to ensure that `Install Jetpack` row appears only when `jetpackConnectionPackageSupport` feature flag is turned on.
- Cleanup localized strings in `SettingsViewController`.
- Reorder enum cases to match the UI of settings screen and add section titles as comments to improve readability.

### Testing instructions
- Make sure that `.jetpackConnectionPackageSupport` feature flag is turned on.
- Launch app and navigate to `Settings` screen and observe that `Install Jetpack` row is visible.
- Go through the strings in `Settings` screen and ensure that they remain unaltered.

### Screenshots


| Before | After |
| --- | --- |
|![Before](https://user-images.githubusercontent.com/524475/142748015-b17f7181-36d7-44fb-ba66-8b8bfc36f6fa.gif) |  ![After](https://user-images.githubusercontent.com/524475/142748060-50992798-ef68-4eb0-99b8-6a246decdeb8.gif)|

---
